dask_ml/ensemble/_blockwise.py

```diff
@@ -62,7 +62,7 @@ def _predict(self, X):
                 dtype=np.dtype(dtype),
                 chunks=chunks,
             )
-        elif isinstance(X, dd._Frame):
+        elif isinstance(X, dd.DataFrame):
```

Comment:
Those aren't equivalent unfortunately, they won't capture Series objects and third party libraries that inherit from _Frame (geopandas for example)

I would create a helper function that does this check